src/main/java/org/ibp/api/rest/derived/DerivedVariableResource.java

```diff
@@ -86,7 +86,7 @@ public ResponseEntity<Set<FormulaVariable>> formulaVariables(
 	}
 
 	@ApiOperation(value = "Count Calculated Traits", notes = "Count the calculated traits (derived traits) in a specified dataset(s)")
-	@PreAuthorize("hasAnyAuthority('ADMIN','STUDIES','MANAGE_STUDIES', 'BROWSE_STUDIES')" + PermissionsEnum.HAS_MANAGE_STUDIES_VIEW)
+	@PreAuthorize("hasAnyAuthority('ADMIN','STUDIES','MANAGE_STUDIES', 'VIEW_STUDIES')" + PermissionsEnum.HAS_MANAGE_STUDIES_VIEW)
```

Comment:
@cuenyad same comment in relation to the enum here

Reply:
done!